cli.test.ts

```
@@ -3,7 +3,14 @@ process.env.AVO_TEST_MODE = 'true';
 
 import fs from 'fs';
 import path from 'path';
-import { describe, it, expect, beforeEach, afterEach } from '@jest/globals';
+import {
+  describe,
+  it,
+  expect,
+  beforeEach,
+  afterEach,
+  jest,
+} from '@jest/globals';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)